popup.js

```diff
@@ -223,6 +223,8 @@ function lookup(hostname, address) {
 			break;
 		case 2:
 			url = `https://www.ip2location.com/${address}`;
+		case 3:
+			url = `https://browserleaks.com/ip/${address}`;
 			break;
```

Comment:
You would need to add a `break` statement after case 2 to prevent it from falling through.